lib/collection_json_serializer/core_ext/symbol.rb

```diff
@@ -5,4 +5,8 @@ def extract_params
 
     params
   end
+
+  def to_constant
+    Object.const_get(self.to_s.split('_').map{|w| w.capitalize}.join)
```

Comment:
Redundant `self` detected.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Pass `&:capitalize` as an argument to `map` instead of a block.